ReleaseNotes.md

```diff
@@ -8,6 +8,32 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 04/07/2023 -
+##### Version microsoft/oms:3.1.6 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.6 (linux)
+##### Version microsoft/oms:win-3.1.6 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.6 (windows)
+##### Code change log
+- Common
+  - Update prometheus.io/scheme default description in configmap
+  - Add support to specify resource endpoint for metrics 
```

Comment:
This change also specific to autonomous.  So update this to "Add support to specify  azure autonomous resource endpoint parameter for the Extension chart for custom metrics "

Reply:
changed